Sources/SwiftGraphQLClient/Client/Core.swift

```diff
@@ -255,7 +255,27 @@ public class Client: GraphQLClient, ObservableObject {
         
         return self.execute(operation: operation)
     }
-    
+
+    /// Executes a query request with given execution parameters.
+    ///
+    /// Note: While this behaves much the same as the published-based
+    /// APIs, async/await inherently does __not__ support multiple
+    /// return values. If you expect multiple values from an async/await
+    /// API, please use the corresponding publisher API instead.
+    ///
+    /// Additionally, due to the differences between async/await and
+    /// Combine publishers, the async APIs will only return a single value,
+    /// even if the query is invalidated. Therefore if you currently 
+    /// rely on invalidation behaviour provided by publishers we suggest
+    /// you continue to use the Combine APIs.
+    public func query(
+        _ args: ExecutionArgs,
+        request: URLRequest? = nil,
+        policy: Operation.Policy = .cacheFirst
```

Comment:
If I'm reading this correctly, the behaviour of this api using the default policy argument of `.cacheFirst`, will only return from the cache in cases where there is a cache hit. When I implemented this in my own project, I had to remove the ability to pass a policy and always use `.networkOnly`. Otherwise it would have to be an async stream.

I know there's a comment on this api, but perhaps the default policy should at least be `.networkOnly`

Reply:
Oh JUST as I merged hahaha, but thank you I'll add this as a patch, I completely missed this, but I agree the change.